qemu/tests/virtio_mem_dynamic_memslots_with_migration.py

```diff
@@ -0,0 +1,50 @@
+import time
+
+from virttest import error_context
+
+from virttest.utils_misc import normalize_data_size
+from provider import virtio_mem_utils
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Virtio-mem dynamic memslots with migration test
+    1) Boot VM
+    2) Resize virtio-mem device
+    3) Do migration
+    4) Check virtio-mem size in dst host
+    5) Validate the virtio-mem device now has the correct number of memslots
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+
+    requested_size_vmem_test = params.get("requested-size_test_vmem0")
+    total_memslots = params.get_numeric("total_memslots", 4)
+    mem_object_id = params.get("mem_devs", "vmem0")
```

Comment:
This parameter is must to boot the vm for this case.
Considering only one mem dev, suggest `params['mem_devs'] `

Reply:
@yanan-fu code updated !